### DISCUSSION TITLE
part-ii/2019-129/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#215] [Question comments]
Part II 2019 Representation Theory: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2019-129/

**Comment by Kiolt** [comment]
<details>
<summary>Part (iv).</summary>

Use all the information about the group to compute that character inner product for the given 4-dim representation to show that order is 32. After that we have full information about conjugacy classes, and we get the 4-dim representation + 16 1-dim representations.

</details>
